test/model/ai/evaluation/BoardEvaluatorTest.java

```diff
@@ -8,25 +8,52 @@
 
 import model.Board;
 import model.PieceColor;
-import model.ai.evaluation.BoardEvaluatorInterface;
-import model.ai.evaluation.KingCountEvaluator;
-import model.ai.evaluation.PawnCountEvaluator;
-import model.ai.evaluation.PlainBoardEvaluator;
 
 public class BoardEvaluatorTest {
 
     @Test
+    public void testBackRowEvaluator() {
```

Comment:
For tests, each concrete class should have a test class that tests the specific details of that class. For abstract classes, the only thing that should be tested is the implemented methods on that abstract. Could you refactor the specific evaluator tests into their own test classes, please?